packages/instantsearch.js/src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -261,32 +261,31 @@ export default (function connectInfiniteHits<
     };
 
     const getShowPrevious =
-      (helper: Helper): (() => void) =>
+      (
+        helper: Helper,
+        getCachedHitsFn: () => InfiniteHitsCachedHits<THit>
```

Comment:
```suggestion
        getCachedHits: () => InfiniteHitsCachedHits<THit>
```

`get` already implies it's a function